### ISSUE TITLE
Uncaught TypeError: this.editor.buffer.updateCachedDiskContents is not a function

### ISSUE THREAD

**Issue by hperrin** [lwblackledge/file-watcher#27]
[Enter steps to reproduce:]

0. Be on Atom Beta.
1. Open a file.
2. Change the file on disk.
3. Click the reload button on the file-watcher dialog.

**Atom**: 1.19.0-beta1 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.4
**Thrown From**: [file-watcher](https://github.com/lwblackledge/file-watcher) package 1.1.0

### Stack Trace

Uncaught TypeError: this.editor.buffer.updateCachedDiskContents is not a function

```
At /Users/hperrin/hperrin-configs/atom/packages/file-watcher/lib/file-watcher.coffee:88

TypeError: this.editor.buffer.updateCachedDiskContents is not a function
    at FileWatcher.module.exports.FileWatcher.forceReload (/packages/file-watcher/lib/file-watcher.coffee:88:22)
    at FileWatcher.module.exports.FileWatcher.confirmReload (/packages/file-watcher/lib/file-watcher.coffee:103:8)
    at /packages/file-watcher/lib/file-watcher.coffee:62:10)
    at emitTwo (events.js:106:13)
    at StatWatcher.emit (events.js:191:7)
    at StatWatcher._handle.onchange (fs.js:1376:10)
```

### Commands

```
```

### Non-Core Packages

```
atom-autocomplete-php 0.25.6 
atom-material-ui 2.0.4 
atom-typescript 11.0.4 
autoclose-html 0.23.0 
busy-signal 1.4.3 
color-picker 2.2.5 
emmet-simplified 2.5.0 
file-icons 2.1.8 
file-watcher 1.1.0 
git-plus 7.9.3 
gl-light-syntax 3.0.0 
highlight-selected 0.13.1 
intentions 1.1.2 
javascript-snippets 1.2.1 
line-diff-details 1.8.0 
linter 2.2.0 
linter-csslint 1.3.4 
linter-eslint 8.2.1 
linter-phpcs 1.6.7 
linter-tidy 2.3.1 
linter-ui-default 1.6.2 
minimap 4.28.2 
minimap-git-diff 4.3.1 
minimap-highlight-selected 4.6.1 
minimap-linter 2.0.0 
pigments 0.39.1 
resize-indent 0.3.0 
slack-ui 0.8.0 
spacegray-light-neue-ui 0.1.7 
tab-switcher 1.5.4 
todo-show 1.11.0 
tree-view-git-status 1.4.0 
```

**Comment by lwblackledge** [comment]
Thank you, looks like the editor rewrite removed some of the low level Atom calls that I was using.
The new editor fix may resolve the mounted file system reload issue but I'll need to test before pushing a pre-release.

**Comment by jainanshul** [comment]
Is there an update on this issue?

**Comment by lwblackledge** [comment]
This should be resolved in the latest update v1.2.4
